code/API_definitions/qod-api.yaml

```
@@ -216,6 +216,79 @@ paths:
           $ref: "#/components/responses/Generic503"
       security:
         - apiKey: []
+  /qos-profiles:
+    get:
+      tags:
+        - QoS Profiles
+      summary: "Get All QoS Profiles"
+      description: |
+        Returns all QoS Profiles that match the given criteria.
+        If no criteria is given, all QoS Profiles are returned.
+      operationId: getQosProfiles
+      parameters:
+        - name: name
+          in: query
+          description: QoS Profile name
+          schema:
+            type: string
+          required: false
+        - name: status
+          in: query
+          schema:
+            $ref: '#/components/schemas/QosProfileStatusEnum'
+          required: false
+      responses:
+        "200":
+          description: Contains information about QoS Profiles
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/QosProfile"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/QosProfilesNotFound404"
+        "500":
+          $ref: "#/components/responses/QoSProfile500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+  /qos-profiles/{qos-profile-name}:
```

```suggestion
  /qos-profiles/{name}:
```

Better as the name of the property, with the description, the context is undestood